src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/NewDataMigrationSelectedDB.cs

```diff
@@ -0,0 +1,114 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="NewDataMigrationSelectedDB.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Management.DataMigration.Models;
+
+namespace Microsoft.Azure.Commands.DataMigration.Cmdlets
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmDataMigrationSelectedDB", SupportsShouldProcess = true, DefaultParameterSetName = SqlServerSqlDbParameterSet), OutputType(typeof(MigrateSqlServerSqlDbDatabaseInput))]
+    [Alias("New-AzureRmDmsSelectedDB")]
+    public class NewDataMigrationSelectedDB : DataMigrationCmdlet
+    {
+        private const string SqlServerSqlDbParameterSet = "MigrateSqlServerSqlDb";
+        private const string SqlServerSqlDbMiParameterSet = "MigrateSqlServerSqlDbMi";
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the source database.",
+            ParameterSetName = SqlServerSqlDbMiParameterSet
+        )]
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the source database.",
+            ParameterSetName = SqlServerSqlDbParameterSet
+        )]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the target database.",
+            ParameterSetName = SqlServerSqlDbMiParameterSet
+        )]
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the target database.",
+            ParameterSetName = SqlServerSqlDbParameterSet
+        )]
+        [ValidateNotNullOrEmpty]
+        public string TargetDatabaseName { get; set; }
+
+        [Parameter(
+            ParameterSetName = SqlServerSqlDbMiParameterSet,
+            Mandatory = true,
+            HelpMessage = "Set migration type to SQL Server to SQL DB MI Migration."
+        )]
+        [ValidateNotNullOrEmpty]
```

Comment:
@hitenjava nit: you don't need the `ValidateNotNullOrEmpty` attribute for a `SwitchParameter`

Reply:
Done

Reply:
I still see the attribute.

Reply:
Fixed